C-Plus-Plus/cp/Tug_of_water.cpp

```diff
@@ -0,0 +1,113 @@
+#include<iostream>
```

Comment:
Question is :: union of two unsorted arrays Issue
and you have submitted the code of :: Tug_of_war ??

please check again everything :)